src/templates/next/layouts/Homepage/Homepage.stories.tsx

```diff
@@ -32,6 +44,10 @@ Default.args = {
       siteNavItems: [],
     },
     lastUpdated: "1 Jan 2021",
+    search: {
+      type: "searchSG",
+      clientId: "7946e346-993e-41c7-bd81-26a3999dc3f4",
```

Comment:
should this be in env vars?

Reply:
Hmm this one is public info (took it from MFA's website), so I think can hardcode it here?